README.md

```diff
@@ -1,20 +1,21 @@
-This is a layer to provide ROS Indigo Igloo in an OpenEmbedded Linux system.
-It provides a stable cross-compilation build system for many common ROS packages. 
-Currently, this layer is still under continuous development.
+# Branch `legacy`
+
+This branch contains the legacy layer version 1 implementation of `meta-ros`
+that provides ROS Indigo Igloo in an OpenEmbedded Linux system. It supplies a
+cross-compilation build system for many common ROS packages. It is no longer
+under active development, but will be retained until it is no longer needed by
+the community.
```

Comment:
There are also some initial bits from ROS 2 Ardent Apalone from https://github.com/ros/meta-ros/pull/538

Maybe worth mentioning that this legacy branch might better support older Yocto releases as well. Or are we going to release and maintain what's currently in morty-draft branch as well?

Reply:
> There are also some initial bits from ROS 2 Ardent Apalone from #538

Please suggest what should be added to the sentence for this to indicate this.

> Maybe worth mentioning that this legacy branch might better support older Yocto releases as well. 

Again, please suggest wording to indicate this.

> Or are we going to release and maintain what's currently in morty-draft branch as well?

Will there be any demand for support for older Yocto releases now that support for thud is available?